101-application-gateway-waf/azuredeploy.parameters.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworkName": {
+      "value": "yourvnetname"
+    },
+    "vnetAddressPrefix": {
+      "value": "10.0.0.0/16"
+    },
+    "subnetName": {
+      "value": "yoursubnet"
+    },
+    "subnetPrefix": {
+      "value": "10.0.0.0/24"
+    },
+    "applicationGatewayName": {
```

Comment:
Check with Amit once if there is some consistence requirement here. In most of our templates Vnet name, subnet name and application gateway name are not parameters (the names are hardcoded).
Personally I like them better as parameters.